### PR TITLE
Notification and Message add className and style

### PR DESCRIPTION
**Pull request by lixiaoyang1992**
Notification and Message add className and style

#5893

### CODE REVIEW

**Review by xueqingxiao — APPROVED** [review]

File: components/message/index.en-US.md [hunk 1]
```diff
@@ -16,12 +16,12 @@ Display global messages as feedbacks to user operations.
...
-- `message.info(content, duration, onClose)`
-- `message.warning(content, duration, onClose)`
-- `message.warn(content, duration, onClose)` // alias of warning
-- `message.loading(content, duration, onClose)`
+- `message.success(content, duration, onClose, className, style)`
+- `message.error(content, duration, onClose, className, style)`
+- `message.info(content, duration, onClose, className, style)`
+- `message.warning(content, duration, onClose, className, style)`
+- `message.warn(content, duration, onClose, className, style)` // alias of warning
+- `message.loading(content, duration, onClose, className, style)`
```
Comment (afc163) [inline components/message/index.en-US.md]: `message.info(<div className="" style={{ .... }}></div>)` will be enough.

Comment (afc163) [inline components/message/index.en-US.md]: The long arguments is a disaster..

**Review by afc163 — CHANGES_REQUESTED** [review]
Don't change message cause arguments is too long.

**Comment by lixiaoyang1992** [comment]
fixed. Now only Notification has been changed.

**Comment by benjycui** [comment]
新增 feature 应该同时更新 API 文档，这次我补上，以后注意吧。

**Pull request merged**